lms/templates/courseware/progress.html

```diff
@@ -29,181 +29,172 @@
+    ${progress_graph.body(grade_summary, course.grade_cutoffs, "grade-detail-graph", not course.no_grade, not course.no_grade) | h}
 </script>
 </%block>
 
-
 <%include file="/courseware/course_navigation.html" args="active_page='progress'" />
 
 <main id="main" aria-label="Content" tabindex="-1">
     <div class="container">
-      <div class="profile-wrapper">
-        <div class="course-info" id="course-info-progress" aria-label="${_('Course Progress')}">
-          % if staff_access and studio_url is not None:
-            <div class="wrap-instructor-info">
-              <a class="instructor-info-action studio-view" href="${studio_url | h}">${_("View Grading in studio")}</a>
-            </div>
-          % endif
-
-          <header class="progress-certificates">
-            <h1 class="progress-certificates-title">${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email) | h}</h1>
-          </header>
-        <div class="wrapper-msg wrapper-auto-cert">
-          %if certificate_data:
-            <div id="errors-info" class="errors-info"></div>
-            %if passed:
-            <div class="auto-cert-message" id="course-success">
-              <div class="has-actions">
-                  <% post_url = reverse('generate_user_cert', args=[unicode(course.id)]) %>
-                  <div class="msg-content">
-                    <h2 class="title">${certificate_data.title | h}</h2>
-                    <p class="copy">${certificate_data.msg | h}</p>
-                  </div>
-                  <div class="msg-actions">
-                    %if certificate_data.cert_web_view_url:
-                    <a class="btn" href="${certificate_data.cert_web_view_url | h}" target="_blank" title="${_('View certificate in a new browser window or tab.')}">
-                      ${_("View Certificate")}
-                    </a>
-                    %elif certificate_data.cert_status == CertificateStatuses.downloadable and certificate_data.download_url:
-                    <a class="btn" href="${certificate_data.download_url | h}" target="_blank" title="${_('PDF will open in a new browser window or tab.')}">
-                      ${_("Download Your Certificate")}
-                    </a>
-                    %elif certificate_data.cert_status == CertificateStatuses.requesting:
-                      <button class="btn generate_certs" data-endpoint="${post_url | h}" id="btn_generate_cert">${_('Request Certificate')}</button>
+        <div class="profile-wrapper">
+            <section class="course-info" id="course-info-progress">
+                <h3 class="hd hd-3 progress-certificates-title">
+                    ${_("Course Progress for Student '{username}' ({email})").format(username=student.username, email=student.email) | h}
+                </h3>
+
+                % if staff_access and studio_url is not None:
```

Comment:
This is a nit given that it is not learner-facing, but the placement of this "View Grading in studio" button is a bit odd (below the header for the student).